recipes/thriftpy/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py build_ext
```

Comment:
In the past I've had some weird inconsistencies with this library when i built it locally.  It _shouldn't_ be required but just in case